pokebattle/urls.py

```diff
@@ -12,6 +12,7 @@
     url(r'^admin/', admin.site.urls, name='admin'),
     url(r'^admin/statuscheck/', include('celerybeat_status.urls')),
     url(r'^jsreverse/$', django_js_reverse.views.urls_js, name='js_reverse'),
+    url('', include('social_django.urls', namespace='social')),
```

Comment:
I think it is a good idea to add an URL pattern here like `r'^social/'`. Defining a pattern for every URL or namespace is always a good idea, so we avoid conflicts in the future.

Also, you won't have problems changing this because your button is using the `url` templatetag with namespace.